packages/frontend/apps/electron/src/main/protocol.ts

```diff
@@ -41,38 +13,34 @@ function isNetworkResource(pathname: string) {
   return NETWORK_REQUESTS.some(opt => pathname.startsWith(opt));
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +12]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->is it possible to detect the request by file existence instead of hardcoded url prefix?<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/8464?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
URLs like `/workspace/xxx/xxx` need to fall back to `index.html`.
Is there a reliable way to determine which URLs require a fallback?

Reply:
unfortunately no.